### DISCUSSION TITLE
How do I implement importing a header value into a template?

### DISCUSSION THREAD

**Discussion by minwoo-yu** [windingwind/zotero-better-notes#760] [Q&A] [answered]
I have a question for acquiring header value in the template file.
If I modified header in somewhere such as obsidian, can I get that header value in the template?

For example, here is my ExportMDFileHeaderV2

```javascript
${{
  let header = {};
  header.tags = noteItem.getTags().map((_t) => _t.tag);
  header.institutions = noteItem.getTags().map((_t) => _t.tag);
  return JSON.stringify(header);
}}$
``` 
And I will add the tags & institutions value in my obsidian synced file.
After that, can I then display those header.tags and header.institutions values in my template file?

I'm confused because I'm not familiar with this language.

**Comment by windingwind** [comment]
What it the `template file`?

**Comment by minwoo-yu** [comment]
The intent of the question is 

```html
<ul>
  <li>📑<b>Publication: </b>${topItem.getField("publication")}</li>
  <li>🗓️<b>Year: </b>${topItem.getField("year")}</li>
</ul>
``` 
If I wrote this [item] Example Template file,
I want to show some additional header values, such as Institution and the tags I added to the header.

**Reply by windingwind** [reply]
To clarify, there is no such `template file`, which is confusing. We only have Zotero note, BN template, and Markdown file.

Sorry, I still do not understand what you are looking for:

> I want to show (where to show?) some additional header values, such as Institution and the tags I added to the header (do you mean you want to use these yaml header parameters in markdown, or do you want to update the item metadata in Zotero from your modifications in markdown through the syncing pipeline?).

**Reply by minwoo-yu** [reply]
Ah BN templete is what I'm trying to modify.

**Reply by windingwind** [reply, marked as answer]
BN template generates note content, while yaml header is only for export to Markdown. I still do not understand what you want. Please answer directly:

> I want to show

(where to show?)

> some additional header values, such as Institution and the tags I added to the header

(do you mean you want to use these yaml header parameters in markdown, or do you want to update the item metadata in Zotero from your modifications in markdown through the syncing pipeline?).

**Reply by minwoo-yu** [reply]
I think the latter is closer to the option I want.

**Reply by windingwind** [reply]
It's the third time I have asked you to answer my inquiries directly. Since you only do casual answers, I will only do casual help.

Please show basic respect to those who volunteer their time to help.

**Reply by minwoo-yu** [reply]
I apologize if I came across as rude.
It was never my intention to do so, and I'm always grateful for this project.
I think the answer you gave above is probably close to the answer.
Once again I apologize..

**Comment by windingwind** [comment]
Please see #527. It's a planned new feature.

I'll close this QA as duplicated
